worker-languagedetection-container/testcases/automated/CAF_4001/limbu.yaml.testcase

```diff
@@ -36,43 +36,43 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "lif"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "100"
+        - data: "76"
       DetectedLanguage1_Name:
         action: "replace"
         values:
-        - data: "LIMBU"
+        - data: "UNKNOWN"
```

Comment:
Significant change here, cannot identify limbu anymore. If it is not supported then remove this testcase and make sure that limbu is not referred to as a supported language.